t/17-build_tagging.t

```diff
@@ -355,4 +356,24 @@ subtest 'content negotiation' => sub {
       ->content_type_is('application/json;charset=UTF-8');
 };
 
+subtest 'test whether tags with datetime work, too' => sub {
+    create_job_version_build("Factory", "2023-03-24 14:33:39+01:00");
+    post_comment_1001 'tag:"2023-03-24 14:33:39+01:00":important:RC3';
+    $t->get_ok('/group_overview/1001')->status_is(200);
+    my @tags = $t->tx->res->dom->find('.tag')->map('text')->each;
+    is(scalar @tags, 7, 'seven builds tagged');
+    is($tags[0], 'RC3', 'tag description shown');
```

Comment:
You could also check whether the tooltip (title attribute) is correct (which should be "important"). The same counts for the other subtest.

Reply:
Isn't this checked in other sub tests?

Reply:
I just thought it wouldn't hurt in that subtest as well.